doc/code/qml_measurements.rst

```diff
@@ -11,8 +11,35 @@ qml.measurements
     See the :doc:`main measurements page <../introduction/measurements>` for
     details on available measurements.
 
+
+.. automodule:: pennylane.measurements
+
+Top-level measurement functions
```

Comment:
if you set these manually, I don't think you can use the automodapi thing below, because now there's an [empty Functions category](https://xanaduai-pennylane--4750.com.readthedocs.build/en/4750/code/qml_measurements.html#functions)

Reply:
not using automodapi makes this change pretty notable. are we sure we want to do this?

Reply:
I just feel like the more we have to remember to update manually, the less likely things are to be updated. The empty Functions category seemed like the lesser of two evils to me - but I can change it if we hate the empty header.

Reply:
Based on offline discussion we are currently:
1. Going ahead with having the top-level imports link to their top-level import documentation, so in the future the `qml.measurements` page will need manual updating if new measurement functions are added
2. Accepting the empty "Functions" header in exchange for less room for human error in keeping the rest of the `qml.measurements` docs updated